src/pybind/mgr/devicehealth/module.py

```diff
@@ -0,0 +1,257 @@
+
+"""
+Device health monitoring
+"""
+
+import errno
+import json
+from mgr_module import MgrModule, CommandResult
+import rados
+from threading import Event
+from datetime import datetime, timedelta, date, time
+
+TIME_FORMAT = '%Y%m%d-%H%M%S'
+
+class Module(MgrModule):
+    OPTIONS = [
+        { 'name': 'active' },
```

Comment:
Not sure about giving modules on/off switches -- wouldn't we just enable/disable the module?

Reply:
The reason is that the module implements a bunch of CLI commands to do things manually and also has (in this case) automatic background scraping.  Perhaps 'active' should be 'background_scraping' true/false instead

Reply:
Ah -- yeah, maybe just "enable_monitoring" or similar?